### PR TITLE
Linking landing app with Client and Professional apps

### LINKED ISSUE #116: Link homepage to Profile page
Link homepage to Profile page, for Professional or Client users

### PR DESCRIPTION
**Pull request by guybeckenstein**
# Resolves #116 

Client user bar

## Header navigation bar 
I made a modification to the code.
Now the user bar (right top corner) has a menu for:
1. guest user
2. Client user
3. Professional user

If a Client or a Professional presses "Profile" button on the menu, they will be redirected to their Profile page.

For making this functionality it was needed to add _urls.py **namespace**_, so I updated `urls.py`

Now the search bar is visible only to logged-in users, not guests.

## Tests
I tested that guests cannot see the **search bar**, and see **only specific options** in the user menu bar.
In the near future, after log in is added, I will add tests for logged in professional and logged in client **Profile** URLs

## Footer
Also added website footer. We dont need SVG files in static we can just use plain HTML code for SVG

## Other
1. Removed `filters` section from homepage. It is too complicate to implement
2. `margin-top` of **learn_more.html** is fit to **homepage.html**
3. Removed `tests.py` because we don't need it

### CODE REVIEW

**Review by idosinger98 — APPROVED** [review]
LGTM

File: landing/tests.py [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# from django.test import TestCase
```
Comment (idoyek) [inline landing/tests.py:1]: This file is empty, if you don't use it please remove it

**Review by idoyek — CHANGES_REQUESTED** [review]

**Review by TalReinfeld — APPROVED** [review]
LGTM

File: landing/tests.py (hunk 1 above)
Comment (guybeckenstein) [reply landing/tests.py:1]: > 

It is in the PR because I deleted it

**Review by idoyek — APPROVED** [review]
LGTM

File: landing/views.py [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 
 
 def homepage(request):
+    user = request.user
+    user_id = None
+
+    if user.is_authenticated:
+        profile = get_object_or_404(Profile, user_id=user)
+
+        if profile.user_type == 'C':
```
[the remark this answers is not in the document]
Comment (guybeckenstein) [reply landing/views.py]: > Not necessary but, would be useful to have an [ENUM](https://docs.python.org/3/library/enum.html) for these `user_type`s?

Resolved

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**